### DISCUSSION TITLE
What is the minimum mongodb version ?

### DISCUSSION THREAD

**Discussion by franckmartin** [payloadcms/payload#2329] [Q&A] [answered]
Hi !

Is there a minimum MongoDB version to run Payload CMS?
Our current MongoDB server is running on version 4, is this likely to cause problems?

Thanks

**Comment by DanRibbens** [comment, marked as answer]
Hi @franckmartin!
You should be able to use MongoDB 4+. I had been developing Payload against v4 up until about 6 months ago and we try keep our Mongo API reliance simple to avoid compatibility issues with DocumentDB / Azure Cosmos DB.

Let me know if you run into anything bizare. The most complex queries come from our reliance on aggregations used in managing [versions ](https://payloadcms.com/docs/versions/overview)if you're concerned, I would start there before diving in too deep.

**Comment by franckmartin** [comment]
Thank you so much for your quick answer @DanRibbens!
We'll give it a try and let you know if we run into anything weird.
